java/src/jmri/jmrit/vsdecoder/VSDecoderManager.java

```diff
@@ -805,11 +802,11 @@ public void reporterPropertyChange(PropertyChangeEvent event) {
                             // look for additional geometric layout information
                             if (geofile_ok) {
                                 Reporter rp = (Reporter) event.getSource();
-                                int new_rp = Integer.parseInt(rp.getSystemName().substring(2)); // ??? connection prefix 3 signs? VSDGeoFile checks for non-numeric part, e.g. "IR7a"
+                                int new_rp = Integer.parseInt(rp.getSystemName().substring(2));
```

Comment:
This code looks like it has a bug, since the system prefix can be two or more letters. For example LT and L2T for LocoNet turnouts.

Use Manager.getSystemSuffix() instead of .substring().

https://www.jmri.org/JavaDoc/doc/jmri/Manager.html#getSystemSuffix(java.lang.String)

Reply:
Done, thanks.